libs/core/src/lib/grid-list/components/grid-list-item/grid-list-item.component.html

```diff
@@ -70,7 +70,7 @@
             ></span>
 
             <button
-                class="fd-grid-list__action-button"
+                [ngClass]="_actionButtonClass"
```

Comment:
Here is some information https://github.com/SAP/fundamental-ngx/pull/6050#discussion_r678106147

Reply:
While I see why you did it this way it's an approach that we have not yet agreed on. This will lead to people starting asking why it's done this way in one component and not in other. When we notice something that can be optimized we create an issue for this, set a quick meeting to discuss it and if agreed on using it we change the whole library. This will avoid having questions like the ones added by Andrii on the same PR regarding `readonly` :)

Reply:
Reverted, as `[ngClass]` is not a recommended approach anymore. 

Angular's `[class]` binding is not possible here, as `fd-button` has its own input field `class` which OK only with the strings, not with arrays/objects/etc.